tech-far-hub/src/components/tfh.scss

```diff
@@ -16,4 +18,40 @@
   .usa-footer__social-heading {
     color: #fff;
   }
+  .usa-footer__nav {
+    padding-top: 0;
+    padding-bottom: 2rem;
+  }
+  .usa-footer__primary-content {
+    line-height: 1.3;
+    display: block;
+  }
+}
+
+// TODO: Fix this to use the tokens defined by uswds
+.tfh-next-link {
```

Comment:
Agreed on the comment of using tokens. I feel they capture 95% of use cases. But then there are times when I have to use straight CSS as well